frontend/src/pages/FileUploadPage/components/FileDropzone.tsx

```diff
@@ -11,12 +11,11 @@ import styles from "../FileUpload.module.css";
 
 function FileDropZone(): ReactElement {
     const rollbar = useRollbar();
+    rollbar.info("Reached dropzone component");
 
     const { handleUpload } = useUpload();
```

Comment:
const { handleUpload, error: uploadError } = useUpload();

then you can print uploadError (or log it)